### PR TITLE
CLI Auto login

### PR DESCRIPTION
**Pull request by j16r**
Add an environment variable ADMIN_CREDENTIALS_FILE which allows setting a file for CLI authentication.
If the file exists, any command that requires authentication will automatically log in.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add `ADMIN_CREDENTIALS_FILE` configuration variable
+
+This variable defaults to `$ROOT_DIR/apicredentials` and when defined / the
```
Comment (connorwstein) [inline docs/CHANGELOG.md:14]: Isn't it `$ROOT/apicredentials` ?

**Review by connorwstein — COMMENTED** [review]
Should be pretty easy to add an auto-login test to remote_client_test? I think we could actually just remove like one (or all) of the `mustLogin` calls and that would suffice

**Comment by j16r** [comment]
> Should be pretty easy to add an auto-login test to remote_client_test? I think we could actually just remove like one (or all) of the `mustLogin` calls and that would suffice

Uhhh... turns out they all work without mustLogin? ... investigating more

**Comment by j16r** [comment]
`mustLogin` is no longer needed because all Applications now use a mock HTTP client and are primed with a logged in session. Once https://github.com/smartcontractkit/chainlink/pull/4057 is merged, I'll make a test specifically with this PR that does not have a primed session.

**Review by connorwstein — APPROVED** [review]

**Pull request merged**